configure.ac

```diff
@@ -475,6 +477,11 @@ dnl check for structures
 
 dnl check for compiler characteristics
 
+# make all symbols hidden by default
+AC_LANG_POP([Fortran])
```

Comment:
We need to pop the Fortran AC language before running the OPAL C scripts.  Perhaps this makes more sense to put at the end of the ENABLE_FORTRAN check.  I made that change in the latest commit.  Alternatively, we might be able to move the OPAL_CHECK_VISIBILITY to somewhere before the fortran check, but either way I think we should probably pop Fortran.